server/index.js

```diff
@@ -19,6 +19,24 @@ const db = new pg.Pool({
 
 app.use(express.json());
 
+app.get('/api/other-accounts/', (req, res, next) => {
+  const sql = `
+    select "username"
+    from "accounts"
+  `;
+  db.query(sql)
+    .then(result => {
+      if (!result.rows[0]) {
+        res.status(404).json({
+          error: 'No users found'
+        });
```

Comment:
It's ok if the list is empty. We usually reserve 404 responses for when the client wants something specific that we don't have. But "all" might mean "none". That's fine.